tests/Microsoft.Identity.Web.Test/CacheExtensionsTests.cs

```diff
@@ -206,17 +238,32 @@ private enum CacheType
                         services.AddDistributedMemoryCache();
                         services.AddLogging(configure => configure.AddConsole())
                         .Configure<LoggerFilterOptions>(options => options.MinLevel = Microsoft.Extensions.Logging.LogLevel.Warning);
+
+                        if (disableL1Cache)
+                        {
+                            services.Configure<MsalDistributedTokenCacheAdapterOptions>(options =>
+                            {
+                                options.DisableL1Cache = true;
+                            });
+                        }
                     });
                     break;
             }
 
             var result = await confidentialApp.AcquireTokenForClient(new[] { TestConstants.s_scopeForApp })
                 .ExecuteAsync().ConfigureAwait(false);
 
+            tokenHandler.ReplaceMockHttpMessageHandler = null;
```

Comment:
Please resolve the GitHub Actions warnings. Id.Web uses nullable. Have a look at https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving